### ISSUE TITLE
miniconda3 v4.13/4.14 binary/container releases missing ?

### ISSUE THREAD

**Issue by davidandreoletti** [conda/conda#11727]
### Checklist

- [X] I added a descriptive title
- [X] I searched open requests and couldn't find a duplicate

### What is the idea?

Conda v4.13 and v4.14 have been [released](https://github.com/conda/conda/blob/main/CHANGELOG.md), yet miniconda3 v4.13 / v4.14 have not been released (nor miniconda3 container images)

### Why is this needed?

Users needing improvements from release v4.13 onwards.

### What should happen?

miniconda package and container should be released.

### Additional Context

N/A

**Comment by kenodegard** [comment]
Duplicate of #11728 

We are waiting on the conda 4.14 package to become available on the defaults channel. As for the miniconda installer, they are not created per conda release, but we can probably expect one to be made for conda 4.14.

**Comment by davidandreoletti** [comment]
@kenodegard miniconda is conda with less default starting packages.

If I may, where can I find info on the miniconda installer / container release policy if there are not updated per conda release

**Comment by davidandreoletti** [comment]
Understood.

@kenodegard [Containers with miniconda3 installed from anaconda.com ](https://hub.docker.com/r/continuumio/miniconda3).

**Comment by kenodegard** [comment]
Docker/AWS images are also updated in sync with Anaconda/Miniconda releases.
